robinson/ronbinson.cpp

```diff
@@ -0,0 +1,69 @@
+#include <fstream>
+
+using namespace std;
+
+ifstream input("robinson.in");
+ofstream output("robinson.out");
+
+int m, n, a[21][21], c, l;
```

Comment:
The declaration of Arrays and matrixes keep on separated lines instead of the same line with other simple variables